frontend/src/components/Sidebar/index.tsx

```diff
@@ -1,28 +1,148 @@
-import React, { FC } from "react";
+import * as React from "react";
+
 import "./sideBar.scss";
-import ButtonNav from "../ButtonNav";
+import {
+  Toolbar,
+  Drawer,
+  List,
+  Typography,
+  ListItem,
+  Divider,
+  CssBaseline,
+  AppBar,
+  IconButton,
+  Box,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+
+import { Link } from "react-router-dom";
+/**
+ * @todo Need to make drawerWidth more dynamic
+ */
+//Used for calculcation for body width
+const drawerWidth = 400;
+
+/** The sidebar is the navigation menu located on the left side of the App */
+export const Sidebar = () => {
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+  /**used to toggle mobile and desktop views for the menu */
```

Comment:
Although i don't think there is a script to tell either way, comments should probably start with caps, and end with a full stop.